protostar/starknet/cheatable_cairo_vm.py

```diff
@@ -4,7 +4,7 @@
 
 class CheatableVirtualMachine(VirtualMachine):
     """
-    VirtualMachine with modified step function that captures execution locals of a hint.
+    `VirtualMachine` with modified `step` function that builds cheatcodes created with `DelayedBuilder`.
```

Comment:
Could we please avoid touching this file entirely in this PR? We don't change any logic here, so it would be great to keep this PR separated from this file.